documentation/book/ref-kafka-bridge-http-configuration.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-configuration.adoc
+
+[id='ref-kafka-bridge-http-configuration-{context}']
+= Kafka Bridge Consumer configuration
+
+Kafka Bridge HTTP configuration is set using the properties in `KafkaBridge.spec.http`.
+This property contains the Kafka Bridge HTTP configuration options.
+
+* `host`
+* `port`
```

Comment:
You should mention that the port should not be `8081` since that conflicts with the healthcheck.